modules/strong_ptr.cppm

```diff
@@ -14,11 +14,15 @@
 
 module;
 
+#include <cstdint>
+#include <exception>
+
 #include <cstddef>
 
 #include <array>
 #include <atomic>
 #include <exception>
+#include <memory>
```

Comment:
Please see style ordering for headers https://libhal.github.io/4.2/contributor_guide/style/#s14-headers